mod-kb-ebsco-java/src/test/java/org/folio/KbEbscoApiEurekaTests.java

```diff
@@ -16,7 +16,7 @@
 
 @FolioTest(team = "spitfire", module = "mod-kb-ebsco-java")
 public class KbEbscoApiEurekaTests extends TestBaseEureka {
```

Comment:
I guess we should rename this file into KbEbscoApiTests and the current KbEbscoApiTests which extends TestBase could be removed